src/replicache.ts

```diff
@@ -416,14 +416,29 @@ export default class Replicache implements ReadTransaction {
     }
 
     // Replay.
+    let onlyUnknownMutators = true;
     for (const mutation of replayMutations) {
       const {original} = mutation;
-      syncHead = await this._replay(
+      const [ref, knownMutator] = await this._replay(
```

Comment:
This seems more complex than I expected. Why can't `_replay` do this check internally to avoid the bookkeeping of `onlyUnknownMutators`?